src/compiler/crystal/tools/doc/html/css/style.css

```diff
@@ -725,6 +725,10 @@ span.flag.lime {
   visibility: visible
 }
 
+.octicon-link {
+  color: black;
+}
```

Comment:
Is this even necessary? It should already inherit the color.

Reply:
It's inside an anchor so it inherits its color. It's not that visible on light mode since the color is dark purple but if you change it to green it looks like this:
![image](https://github.com/crystal-lang/crystal/assets/18014039/b6e41ef4-0fe7-44e4-8c3b-e7eaf1e78e49)

Reply:
Yes, I think it's best to use the same color as the link text.

(It *could* be a different color, but then it would need to be *really* different, not almost the same.)

Reply:
Should it inherit it only for light mode or for both? On dark mode the anchor color being light purple / blurple is more noticeable:

![image](https://github.com/crystal-lang/crystal/assets/18014039/802d414c-c188-4428-af31-e0abb2bc70ea)

Reply:
Yeah, I think it makes most sense to use link color everywhere. I mean... it's a link.